06-building-a-program-start-to-finish/writing-tests/output/spec/InvertedIndexSpec.js

```diff
@@ -0,0 +1,276 @@
+const newIndex = new InvertedIndex();
+
+describe('Tests for the InvertedIndex class', () => {
+
+  describe('Tests for the validateFile method', () => {
+    it('should return error message for non-Array objects', () => {
```

Comment:
Do you mean 
_**Tests for the validateFile method, should return error message for non-Array objects**_
or 
**_validateFile method, should return error message for non-Array objects_**

Please ensure that the language used on your test communicates appropriately. Please address this on all test suites.